pkgs/applications/misc/sampler/default.nix

```diff
@@ -0,0 +1,29 @@
+{ stdenv, buildGoModule, fetchFromGitHub, alsaLib }:
+
+buildGoModule rec {
+  pname = "sampler";
+  version = "v1.0.3";
+
+  src = fetchFromGitHub {
+    owner = "sqshq";
+    repo = "sampler";
+    rev = "${version}";
+    sha256 = "129vifb1y57vyqj9p23gq778jschndh2y2ingwvjz0a6lrm45vpf";
+  };
+
+  modSha256 = "0wgwnn50lrg6ix5ll2jdwi421wgqgsv4y9xd5hfj81kya3dxcbw0";
+
+  subPackages = [ "." ];
+
+  buildInputs = [ alsaLib ];
+
+  meta = with stdenv.lib; {
+    description = ''
```

Comment:
`description` should preferably be single line and without an ending punctuation mark.
Use `longDescription` for multiline descriptions

https://nixos.org/nixpkgs/manual/#sec-standard-meta-attributes